metaflow/plugins/kfp/kfp_constants.py

```diff
@@ -12,6 +12,6 @@
 TASK_ID_ENV_NAME = "TASK_ID_ENV_NAME"
 SPLIT_INDEX_ENV_NAME = "SPLIT_INDEX_ENV_NAME"
 INPUT_PATHS_ENV_NAME = "INPUT_PATHS_ENV_NAME"
-RETRY_COUNT = "RETRY_COUNT"
+RETRY_COUNT = "MF_ATTEMPT"
```

Comment:
Naming change by using `export_mflog_env_vars` over the `retry_count_python` bash command for setting env var.